src/debugger/launch.ts

```diff
@@ -88,6 +89,7 @@ function createExecutableConfigurations(ctx: FolderContext): vscode.DebugConfigu
                 args: [],
                 cwd: folder,
                 preLaunchTask: `swift: Build Debug ${product.name}${nameSuffix}`,
+                env: swiftRuntimePathEnv({}),
```

Comment:
Any reason you are passing `{}` instead of `process.env`?

Reply:
Since this is going to be persisted in `launch.json`, I tend to have the minimal list of paths. Regular Swift executables won’t depend on anything but the Swift runtime (Windows stuffs are always available regardless of `%Path%`). If the user need more, he can change the config manually or through "Test Environment Variables".